File: backEnd/platform-automation/src/main/java/com/mokatest/platform/demos/api/domain/apiEnum/AuthType.java
package com.mokatest.platform.demos.api.domain.apiEnum;

/**
 * 接口鉴权类型
 * <p>
 * NONE    : 无鉴权，不附加任何认证信息
 * BEARER  : Bearer Token，生成 Authorization: Bearer {token}
 * BASIC   : Basic Auth，生成 Authorization: Basic base64(username:password)
 * API_KEY : 自定义 Key，可放到 Header 或 Query
 */
public enum AuthType {
    NONE,
    BEARER,
    BASIC,
    API_KEY
}

File: frontEnd/src/types/domain/api/apiEnum/AuthType.ts
/**
 * 接口鉴权类型枚举
 * 使用字符串值以匹配后端 Jackson 序列化（enum name → "BEARER"）
 */
export enum AuthType {
    NONE = "NONE",
    BEARER = "BEARER",
    BASIC = "BASIC",
    API_KEY = "API_KEY"
}
